addons/im_livechat/report/im_livechat_report.py

```diff
@@ -1,49 +1,65 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, fields, models, tools
+from odoo import fields, models, tools
 from odoo.tools import SQL
 
 
-class Im_LivechatReportChannel(models.Model):
-    """ Livechat Support Report on the Channels """
+class Im_LivechatReport(models.Model):
+    """ Livechat Support Report """
 
-    _name = 'im_livechat.report.channel'
-    _description = "Livechat Support Channel Report"
-    _order = 'start_date, technical_name'
+    _name = "im_livechat.report"
```

Comment:
I wonder if we should avoid renaming the model/table and avoid renaming files, it would make a lot less diff and less upgrade, and the rename brings little gain.

It would be easier to follow if we renamed separately or didn't rename at all.

Reply:
So just keep the "channel" report and drop the operator report.